visualizer/index.html

```diff
@@ -161,6 +171,105 @@
       color: white;
     }
 
+    .semanticsEditor.hidden .header,
+    .semanticsEditor.hidden .body,
+    .semanticsEditor.hidden .save {
+      display: none;
+    }
+
+    .semanticsEditor .header {
+      font-family: Menlo, Monaco, sans-serif;
+      font-size: 12px;
+      display: flex;
+      height: 24px;
+      margin-bottom: 8px;
+    }
+    .semanticsEditor .header .block {
+      padding-top: 5px;
+      padding-bottom: 5px;
+      float: left;
+      margin-left: 5px;
+    }
+    span.name {
+      color: rgb(123, 123, 123);
+      float: left;
+    }
+    textarea.rename {
+      height: 12px;
+      resize: none;
+      outline: none;
+      margin-top: 8px;
+      font: italic 12px Menlo, Monaco, sans-serif;
+      color:  #FF8D00;
+      border: white;
+    }
+
+    .semanticsEditor .body {
+      margin-top: 3px;
+      border: 1px solid #ddd;
+      height: 100px;
+      min-width: 300px;
+      overflow: auto
+    }
+
+    .semanticsEditor .result {
+      padding-bottom: 2px;
+      text-align: center;
+      color: #4CAF50;
+      height: 13px;
+    }
+    .semanticsEditor .result.error {
+      background-color: rgb(255, 215, 222);
+      color: red;
+    }
+
+    .semanticsEditor button.save {
+      width: 100%;
+    }
+    .semanticsEditor button.save:hover {
+      text-decoration: underline;
+    }
+
+    #actions {
+      display: flex;
+      position: fixed;
```

Comment:
Trying to keep the action buttons position fixed, so it won't change while we scrolling the bottom section